src/main/java/com/revature/models/ShopProduct.java

```diff
@@ -20,7 +20,7 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "shop_product_id")
-    private int id;
+    private int shop_product_id;
```

Comment:
This will break other features. Rule of thumb for models: annotations can change, fields cannot